_layouts/main.html

```diff
@@ -7,10 +7,7 @@
 
   <body>
     <div class="container">
-      <section class="sidebar">
-        {% include header.html %}
-        {% include navigation.html %}
-      </section>
+      {% include navigation.html %}
```

Comment:
Nice! I like that we're removing that entire section tag in favor of a more simplified and semantic `nav` tag in this navigation file.